theSrc/scripts/LabeledScatter.js

```diff
@@ -116,14 +116,16 @@ class LabeledScatter {
     const drag_layer = d3.select(this.rootElement).select('.draglayer')
     this.moveDragLayerToBeLast()
 
-    const plot_width = plotly_chart_layout.xaxis._length
-    const plot_height = plotly_chart_layout.yaxis._length
+    const nsewdrag_rect = this.nsewdragRect()
```

Comment:
plotly_chart_layout was incorrect when the axis labels were long, so I'm now using the rect from getBBox on the nsewdrag element